code/modules/food/food/snacks.dm

```diff
@@ -2226,7 +2245,17 @@
 /obj/item/weapon/reagent_containers/food/snacks/candiedapple
 	name = "Candied Apple"
 	desc = "An apple coated in sugary sweetness."
-	icon_state = "candiedapple"
+	icon_state = "candiedapple2"
+	filling_color = "#F21873"
+	center_of_mass = list("x"=15, "y"=13)
+	nutriment_amt = 3
+	nutriment_desc = list("apple" = 3, "sweetness" = 2)
+	bitesize = 3
+
+/obj/item/weapon/reagent_containers/food/snacks/caramelapple
+	name = "Caramel Apple"
```

Comment:
There's some inconsistencies with food names - some of them are `\improper` but some are capitalized like here.